bookwyrm/templates/book/edit/edit_book_form.html

```diff
@@ -275,14 +275,26 @@ <h2 class="title is-4">
                     </div>
                 </div>
 
-                <div class="field">
+                <div class="field{% if form.physical_format.value == 'AudiobookFormat' %} is-hidden{% endif %}" id="toggle-target-pages">
                     <label class="label" for="id_pages">
                         {% trans "Pages:" %}
                     </label>
                     {{ form.pages }}
 
                     {% include 'snippets/form_errors.html' with errors_list=form.pages.errors id="desc_pages" %}
                 </div>
+
+                <div class="field{% if form.physical_format.value != 'AudiobookFormat' %} is-hidden{% endif %}" id="toggle-target-audiobook-play-time">
+                    <label class="label" for="id_audiobook_play_time">
+                        {% trans "Audiobook play time:" %}
+                    </label>
+                    {{ form.audiobook_play_time }}
+                    <p class="help">
+                        {% trans "Can be filled as hours:minutes or as the total run in minutes" %}
+                    </p>
+
+                    {% include 'snippets/form_errors.html' with errors_list=form.audiobook_play_time.errors id="desc_audiobook_play_time" %}
+                </div>
```

Comment:
I like the generalized toggling in `toggleOnSelect`! Because we're toggling two ways (both on render, and reactively with Javascript), this will break this portion of the form for people with Javascript disabled.

It seems there's no good way to react to a dropdown changing with just CSS, so can we fall back to showing both fields at render time, and run the javascript code to hide one of the fields on pageload instead? It's not the best experience to have two fields and only need one, but I think it's better than switching it to an audiobook and not being able to input the length.